test/test_integrate.py

```diff
@@ -270,19 +286,36 @@ def get_terms(bm):
     else:
         assert False
 
+    if theoretical_order == 0.5:
+        levels = (3, 8)
+        ref_level = 10
+    elif theoretical_order == 1.0:
+        levels = (1, 7)
+        ref_level = 13
+    elif theoretical_order == 1.5:
+        levels = (0, 5)
+        ref_level = 13
```

Comment:
Do I understand correctly here that you're only testing two levels for the sake of speed?
If so, looks good to me -- nice idea!